web/src/engine/websites/ScyllaScans.ts

```diff
@@ -0,0 +1,24 @@
+import { Tags } from '../Tags';
+import icon from './ScyllaScans.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as ReaderFront from './decorators/ReaderFront';
+
+const apiUrl = 'https://api.scyllascans.org';
+
```

Comment:
For consistency consider to use an intermediate variable with expressive name as in the other websites
`const cdnUrl = apiUrl;`